src/routes/news/components/_tests_/NewsList.spec.js

```diff
@@ -0,0 +1,209 @@
+// @flow
+
+import * as React from 'react'
+import TestRenderer from 'react-test-renderer'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import configureMockStore from 'redux-mock-store'
+import CityModelBuilder from '../../../../testing/builder/CityModelBuilder'
+import LanguageModelBuilder from '../../../../testing/builder/LanguageModelBuilder'
+import createNavigationScreenPropMock from '../../../../modules/test-utils/createNavigationScreenPropMock'
+import LocalNewsModelBuilder from '../../../../testing/builder/NewsModelBuilder'
+import type {
+  NewsRouteStateType,
+  StateType,
+  LanguagesStateType,
+  CitiesStateType
+} from '../../../../modules/app/StateType'
+import { Provider } from 'react-redux'
+import { render } from '@testing-library/react-native'
+import ErrorCodes from '../../../../modules/error/ErrorCodes'
+import { Text } from 'react-native'
+import { LOADING_TIMEOUT } from '../../../../modules/common/constants'
+import TranslatedNewsListItem from '../NewsListItem'
+import TranslatedWithThemeNewsList from '../NewsList'
+
+const mockStore = configureMockStore()
+jest.mock('react-i18next')
+jest.useFakeTimers()
+
+class MockNewsList extends React.Component<{}> {
+  render () {
+    return <Text>News List</Text>
+  }
+}
+
+class MockList extends React.Component<{}> {
+  render () {
+    return <Text>List</Text>
+  }
+}
+
+describe('News', () => {
+  const [city] = new CityModelBuilder(1).build()
+  const cities = new CityModelBuilder(1).build()
+  const languages = new LanguageModelBuilder(1).build()
+  const language = languages[0]
+  const newsList = new LocalNewsModelBuilder(
```

Comment:
as in previous `localNewsModuleBuilder` lets change `NewsList-Component` to `newsList-component`

Reply:
fixed